Sandbox.ShoppingCart/Models/Product.cs

```diff
@@ -11,5 +11,6 @@ public class Product
         public string Name { get; set; }
         public string Description { get; set; }
         public double Price { get; set; }
+        public double Qty { get; set; }
```

Comment:
Please change Qty to QtyOnHand. As Qty will have another will have a different meaning in the future cart.

Reply:
I changed to "QtyAvailable"